docker/Dockerfile

```diff
@@ -1,13 +1,13 @@
-# VERSION 1.10
-# AUTHOR: Subash Canapathy
+# VERSION 1.10.15
```

Comment:
Keep this 1.10 as its common for both. you have the specific version already in the ARG line below.

Reply:
Okay I will revert that back.